### DISCUSSION TITLE
Previewing content items does not display the full page, only the content items are displayed

### DISCUSSION THREAD

**Discussion by PaariMobimp** [OrchardCMS/OrchardCore#16693] [Q&A]
Hi, so I have created some content items and have integrated it with my asp.net core project and its working fine. The issue is when I edit the content item and hit preview, I am getting a new page where only the content items are displayed. I should be getting a full page with the content items inside it.

**Comment by hishamco** [comment]
Please steps to reproduce and screenshot if it's possible

**Reply by PaariMobimp** [reply]
Steps to reproduce:

1. Create a content type 
2. Create a new content item
3. Publish the content item
4. Edit the content item and click on Preview

I have wrapped the orchard content item with my .net core project and upon previewing I should see the full page with content items inside it.

![Expected page upon preview](https://github.com/user-attachments/assets/1af3413d-39d4-4421-bd5a-58dfde77bc73)
Expeced page upon clicking preview

![Actual page upon preview](https://github.com/user-attachments/assets/9e3b982d-b1ff-4c75-92e0-4227ae2916ee)
Actual page that I got upon preview

**Reply by hishamco** [reply]
Could you please show me the published page without a preview

**Comment by PaariMobimp** [comment]
This is the published page without preview. Its working fine. 
![image](https://github.com/user-attachments/assets/34f84dc7-4de2-4c50-8530-7642024c5d46)

**Comment by PaariMobimp** [comment]
What I have noticed is that when i preivew the page,  I am getting a single article element wrapping all my content items. 
![Preview_articleWrap](https://github.com/user-attachments/assets/22ef3d18-861a-4b9a-869a-9316f1b99c49)
